ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward.cpp

```diff
@@ -543,11 +541,11 @@ std::vector<Tensor> ExecuteUnaryBackwardLogit::invoke(const Tensor& grad, const
         std::nullopt,
         output_mem_config);
     grad_result = where(
-        ttnn::eq(status, ttnn::ones_like(input), std::nullopt, output_mem_config), grad_result, std::nanf(""));
+        ttnn::eq(status, 1.0f, std::nullopt, output_mem_config), grad_result, std::nanf(""));
     grad_result = where(
         ttnn::logical_or(
-            ttnn::eq(input, 0.0, std::nullopt, output_mem_config),
-            ttnn::eq(input, 1.0, std::nullopt, output_mem_config),
+            ttnn::eq(input, 0.0f, std::nullopt, output_mem_config),
+            ttnn::eq(input, 1.0f, std::nullopt, output_mem_config),
```

Comment:
int not supported?

Reply:
float scalar only. 
<img width="1043" alt="image" src="https://github.com/user-attachments/assets/9fce9470-c6a8-41b3-a4b1-b7706741604f">